griptape/loaders/web_loader.py

```diff
@@ -13,5 +13,8 @@ class WebLoader(BaseTextLoader):
     )
 
     def load(self, source: str, *args, **kwargs) -> ErrorArtifact | list[TextArtifact]:
-        single_chunk_text_artifact = self.web_scraper_driver.scrape_url(source)
-        return self._text_to_artifacts(single_chunk_text_artifact.value)
+        try:
+            single_chunk_text_artifact = self.web_scraper_driver.scrape_url(source)
+            return self._text_to_artifacts(single_chunk_text_artifact.value)
+        except Exception as e:
+            return ErrorArtifact(f"Error loading text from {source}", exception=e)
```

Comment:
is it always TEXT that's scraped?

Reply:
This is meant to be referring to what the [output type should be](https://github.com/griptape-ai/griptape/blob/dev/griptape/loaders/web_loader.py#L15) (list[TextArtifact]), rather than what it's loading. 

Should it say something more explicit like `Error loading TextArtifact(s) from source`?

Reply:
could just even say `Error loading from source: {source}`. the method is called `load` after all.